rules/review-videos-collaboratively/rule.md

```diff
@@ -115,6 +119,33 @@ Exporting feedback - to download all comments:
 
 ![Figure: Mark comment as done button](frame-edit-3.png "Mark comment as done button")
 
+## Why you shouldn't review videos on delivery platforms
+
+The top video social media are:
+
+* YouTube
+* YouTube Shorts
+* Vimeo
+* Instagram
+* TikTok
+
+Comparison table using delivery platforms over Frame.io for reviewing
+
+| <!--StartFragment-->                | Frame.io | One Drive | Vimeo | YouTube     | YouTube Shorts | Instagram | TikTok              |
+| ----------------------------------- | -------- | --------- | ----- | ----------- | -------------- | --------- | ------------------- |
+| Playback Controls                   | ✅        | ✅         | ✅     | ✅           | ❌              | ❌         | ❌                   |
+| Basic Timecode                      | ✅        | ✅         | ✅     | ✅           | ❌              | ❌         | ❌                   |
+| Advance Timecode with Frame Counter | ✅        | ❌         | ❌     | ❌           | ❌              | ❌         | ❌                   |
+| Unlisted Videos                     | ✅        | ✅         | ✅     | ✅           | ✅              | ❌         | ❌                   |
+| Versioning                          | ✅        | ✅         | ✅     | ❌           | ❌              | ❌         | ❌                   |
+| Timestamped comments                | ✅        | ❌         | ✅     | ✅           | ❌              | ❌         | ❌                   |
+| Project File Management             | ✅        | ✅         | ❌     | ❌           | ❌              | ❌         | ❌                   |
+| Downloadable                        | ✅        | ✅         | ✅     | Low Quality | Low Quality    | ❌         | ❌                   |
+| Draw on Video                       | ✅        | ❌         | ❌     | ❌           | ❌              | ❌         | ❌                   |
+| Export Comments                     | ✅        | ❌         | ❌     | ❌           | ❌              | ❌         | ❌<!--EndFragment--> |
+
+<br>
```

Comment:
Hey @williamgreentreessw 

1. Please remove this HTML from Markdown

Reply:
Removed HDML Tags